helpers_test.go

```diff
@@ -27,7 +27,7 @@ func consistentlyNot(t *testing.T, cond func() bool) bool {
 	return consistently(t, func() bool { return !cond() })
 }
 
-func chanClosed(ch <-chan time.Time) func() bool {
+func chanClosed[T any](ch <-chan T) func() bool {
```

Comment:
This one. ^

I was going to update the `chanReceives` and `structChanReceives` ones, but as far as I can tell there's no "comparable" interface the generic can be constrained to. =/